.github/workflows/ci.yml

```diff
@@ -156,6 +156,22 @@ jobs:
         run: pnpm run lint
       - name: VSCode extension pnpm build
         run: just build-extension
+  plugin:
+    name: Intellij Plugin
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Init Hermit
+        uses: cashapp/activate-hermit@v1
+        with:
+          cache: true
+      - name: Build Cache
+        uses: ./.github/actions/build-cache
+      - name: Install Java
+        run: java -version
+      - name: Build Intellij Plugin
+        run: just build-intellij-plugin
```

Comment:
How long does this take? Copy pasta the `if` statement from other jobs if this is going to take more than a minute or two, so we can keep the main PR loop fast.